test/facade/MonitoringAspect.test.ts

```diff
@@ -0,0 +1,49 @@
+import { SynthUtils } from "@monocdk-experiment/assert";
+import { Stack } from "monocdk";
+import * as apigw from "monocdk/aws-apigateway";
+import * as apigwv2 from "monocdk/aws-apigatewayv2";
+import * as acm from "monocdk/aws-certificatemanager";
+
+import {
+  DefaultDashboardFactory,
+  DashboardRenderingPreference,
+  MonitoringFacade,
+} from "../../lib";
+
+test("snapshot test: MonitoringAspect", () => {
```

Comment:
Even this might grow very long. What about creating one stack in a helper method, and then having one test case for each construct type?